docs/whirlpool/docs/03-Getting Started/03-Provide Liquidity/01-Open Position/02-CLMM Pool.md

```diff
@@ -0,0 +1,77 @@
+---
+sidebar_label: CLMM Pool
```

Comment:
Instead of CLMM I would call it `Concentrated Liquidity Pool`

Reply:
Agree